pkg/api/types.go

```diff
@@ -504,20 +505,31 @@ func (l *VolumeList) Set(value string) error {
 	if len(value) == 0 {
 		return errors.New("invalid format, must be source:destination")
 	}
+
+	var pos int
+	isWindows, _ := regexp.MatchString("^[A-Za-z]:\\.*", value)
+	if isWindows {
+		pos = strings.IndexAny(value[3:], ":") + 3
```

Comment:
I believe I just used IndexAny because that was the first to come to mind. I'll change it to `strings.Index`

The '3' is for the `C:\` that prefixes windows directories. This is so that we avoid that detail. I'll add a comment to clarify.

We are changing it to `strings.Index` to grab the first instance of ':', which will preserve the ':z' flag in the destination directory.